src/Domains/Connectors/DriveCentric/Client.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Kanvas\Connectors\DriveCentric;
+
+use Illuminate\Http\Client\PendingRequest;
+use Illuminate\Support\Facades\Http;
+use Kanvas\Apps\Models\Apps; // Import the Http class
+use Kanvas\Connectors\DriveCentric\Enums\ConfigurationEnum;
+
+class Client
+{
+    public function __construct(
+        protected Apps $app
```

Comment:
@FredPeal you cant rely only on app for drive centric since it a CRM it mean you need to use company to get the store id